packages/store-ui/src/molecules/Modal/Modal.test.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+
+import Modal from './Modal'
+import Button from '../../atoms/Button'
+import Input from '../../atoms/Input'
+
+const modalTestId = 'store-modal'
+
+const TestModal = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const handleOpen = () => {
+    setIsOpen(true)
+  }
+
+  const onDismiss = () => {
+    setIsOpen(false)
+  }
+
+  return (
+    <>
+      <Button testId="trigger" onClick={handleOpen}>
+        OpenModal
+      </Button>
+      <Modal isOpen={isOpen} testId={modalTestId} onDismiss={onDismiss}>
+        <Input testId="first-input" />
+        <Button testId="first-button" />
+      </Modal>
+    </>
+  )
+}
+
+describe('Modal', () => {
+  it('Test data-store-modal', () => {
```

Comment:
I believe it would be good to be more descriptive, for example:
```suggestion
  it('The attribute data-store-modal should be present', () => {
```